packages/cli/src/build/__tests__/basic.ts

```diff
@@ -1211,6 +1211,6 @@ test("importing css fails with a nice error", async () => {
     "src/blah.css": "",
   });
   await expect(build(dir)).rejects.toMatchInlineSnapshot(
-    `[Error: 🎁 @scope/test only .ts, .tsx, .js, .jsx, and .json files can be imported but "./blah.css" is imported in "src/index.js"]`
+    `[Error: 🎁 @scope/test only .ts, .tsx, .js, .jsx, and .json files can be imported but "blah.css" is imported in "src/index.js"]`
```

Comment:
The change is caused by the added `path.relative` call here: https://github.com/preconstruct/preconstruct/pull/492/files#diff-81fa7127f1a6e4820814f89af91be081857155992595217ccf33830d6116e91eR59

Right now `source` is already resolved - I don't know how to trace it back to how it was written in the original source code. If we can't trace it back perhaps we should add `./` to computed relative paths if they happen to be in the same directory as the importer?